src/pybind/cephfs/cephfs.pyx

```diff
@@ -1167,7 +1526,78 @@ cdef class LibCephFS(object):
             raise make_ex(ret, "error in write")
         return ret
 
-    def getxattr(self, path, name, size=255):
+    def truncate(self, path, size):
```

Comment:
Not opposed to adding this, but I'll note that since you're also exposing setattrx/fsetattrx, you don't really need a dedicated truncate/ftruncate functions. They should be equivalent to setattrx(CEPH_SETATTR_SIZE). It might be good to just add setattrx and then implement these functions as wrappers around it. I worry a bit that having two parallel implementations of truncate could lead to subtle behavior differences between the two.

Reply:
Ack. I will incorporate those changes.

Reply:
Done. utimes functions are not added with this patch. Should it be done as part of this patch ? or separate?